api/router.go

```diff
@@ -29,7 +29,7 @@ var routes = map[string]map[string]handler{
 		"/images/viz":                     notImplementedHandler,
 		"/images/search":                  proxyRandom,
 		"/images/get":                     getImages,
-		"/images/{name:.*}/get":           proxyImage,
+		"/images/{name:.*}/get":           getImage,
```

Comment:
@jimmyxian I don't get why do you need a custom handler here, using proxy ensure we have the same behaviour as the docker daemon.